src/lib/Elf.cc

```diff
@@ -73,37 +77,38 @@ Elf::Elf(std::string path, char** imagePointer) {
    */
 
   // Seek to the byte representing the start of the header offset table.
-  uint64_t headerOffset;
-  file.read(reinterpret_cast<char*>(&headerOffset), sizeof(headerOffset));
+  // Holds the program header table's file offset in bytes.  If the file has no
+  // program header table, this member holds zero
```

Comment:
The comment says "this value holds zero". Is this enforced when `file.read` is performed or should we be initialising `e_phoff` to 0 ourselves?

Reply:
Initialised to 0 to be sure